tests/app/v2/notifications/test_post_notifications.py

```diff
@@ -132,95 +81,9 @@ def test_post_sms_notification_with_persistance_in_celery_returns_201(
     assert mocked.called
 
 
-def test_post_sms_notification_uses_inbound_number_as_sender(notify_api, client, notify_db_session, mocker):
```

Comment:
We might want to change this test instead of axing it, by mocking the celery database-task and checking what it receives maybe?